controllers/challenge.js

```diff
@@ -84,7 +84,15 @@ const _ = require('lodash');
           req.flash('errors', { msg: 'Unexpected Error Occurred. Contact Webmaster ASAP.' });
           return res.redirect('/');
         }
+        team.totalCompleted += 1;
         team.score += challenge.points;
+        
+        // Below depleted score by predetermined amount 
+        if(challenge.depletionLeft > 0){
```

Comment:
Shouldn't this be moved before adding to team scores?
What would happen if depletionLeft is 0; in that case do we want to give 0 points to the team who finishes the challenge or do we want to expire the challenge?

Reply:
Let's say we have 60 points on the board for that specific challenge

The first person to get it will receive 60 points it will then see if depletion left is greater than 0 if it is it will deplete both depletion left and depletion by, by the same amount

then the next person to get that challenge (assuming depletionLeft = 10 and depletionBy = 2)
will received 58 points and depletion left will now be 8 and depletion By will stay at 0